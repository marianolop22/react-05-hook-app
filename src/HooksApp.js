import React from 'react'
import PropTypes from 'prop-types'

export const HooksApp = props => {
    return (
        <div>
            <h1>Hola Mundo</h1>
        </div>
    )
}

HooksApp.propTypes = {

}
