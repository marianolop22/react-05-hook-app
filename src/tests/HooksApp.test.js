import '@testing-library/jest-dom';
import React from 'react';
import { shallow } from 'enzyme';

import {HooksApp} from '../HooksApp'

describe ('Prueba de HookApp', ()=> {

    test('debe renderizar HookApp', () => {
        const wrapper = shallow(<HooksApp/>);
        // console.log(wrapper.find('h1').text() )
        expect ( wrapper).toMatchSnapshot();
        expect ( wrapper.find('h1').text()).toBe('Hola Mundo');
    });

});