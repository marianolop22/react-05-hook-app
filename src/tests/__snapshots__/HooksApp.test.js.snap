// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de HookApp debe renderizar HookApp 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
